ports/mimxrt10xx/supervisor/port.c

```diff
@@ -257,8 +257,8 @@ safe_mode_t port_init(void) {
     // enabled. It won't occur very often so it'll be low overhead.
     NVIC_EnableIRQ(SNVS_HP_WRAPPER_IRQn);
 
-    // Reset everything into a known state before board_init.
-    reset_port();
+    // Note that reset_port CANNOT GO HERE, unlike other ports, since we currently rely on it to 
+    // protect never_reset pins per board.
```

Comment:
```suggestion
    // Note that `reset_port` CANNOT GO HERE, unlike other ports, because `board_init` hasn't been run yet, which uses `never_reset` to protect critical pins from being reset by  `reset_port`.
```

Reply:
@tannewt was going to ask you about this - do you think this is the best solution? Or should I move the never_resets somewhere else?

Reply:
This is the best option I can think of.